Click into the grey part of the following cell, then run it by pressing Shift + Enter or clicking 'Cell'-> 'Run Cells' in the above toolbar.

In [1]:
import sys; print('python:\t\t\t', str(sys.version_info[0]) + '.' + str(sys.version_info[1]))
print('modules:')
import numpy;   print('\tnumpy:\t\t', numpy.__version__)
import scipy;   print('\tscipy:\t\t', scipy.__version__)
import pandas;  print('\tpandas:\t\t', pandas.__version__)
import sklearn; print('\tsklearn:\t', sklearn.__version__)

python:			 3.6
modules:
	numpy:		 1.14.3
	scipy:		 1.1.0
	pandas:		 0.23.0
	sklearn:	 0.19.1


The output should be something like this:

`
python:        3.6
modules:
    numpy:		1.14.3
    scipy:		1.1.0
    pandas:		0.23.1
    sklearn:	0.19.1
`

Let your instructor know if the output doesn't match.